test/test_other.py

```diff
@@ -12065,6 +12065,18 @@ def test_pthread_hello(self, args):
   def test_pthread_relocatable(self):
     self.do_run_in_out_file_test('hello_world.c', emcc_args=['-sRELOCATABLE'])
 
+  @node_pthreads
+  def test_pthread_unavailable(self):
+    # Run a simple hello world program that uses pthreads
+    self.emcc_args += ['-sPROXY_TO_PTHREAD', '-sEXIT_RUNTIME']
```

Comment:
Why EXIT_RUNTIME?

Reply:
For now this is needed for all node tests that use `PROXY_TO_PTHREAD`.   I've for a fix in flight in #22445 but I'm still working out some issues.